functions/getSpliting/getSpliting.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @function getSpliting
+ * @param {number} arg1 
+ * @param {number} arg2 
+ * @returns {number} number or empty
```

Comment:
Correct the return instruction according to the condition of the task:
@returns {number | undefined} number or empty